app/scaffold/pkgs/pkgs.go

```diff
@@ -70,11 +99,15 @@ func IsRemote(str string, shorts map[string]string) (expanded string, ok bool) {
 					return "", false
 				}
 
-				return out, true
+				return out, pkgUrl.IsRemoteEndpoint(out)
 			}
 		}
 	}
 
+	if pkgUrl.IsRemoteEndpoint(str) {
+		return str, true
+	}
+
```

Comment:
We check for remoteness _after_ checking for short codes, because strings like `gh:smoores-dev/scaffold` can be valid SCP endpoints, since user and password are optional and `gh` is a valid host name.

In practice this is probably what folks want, anyway: if I set a short code, I would expect anything matching it to be replaced _first_, even if that short code is technically a valid URL itself.